src/api/proto/cloudpb/cloudapi.proto

```diff
@@ -76,6 +77,16 @@ message UpdateUserRequest {
   google.protobuf.BoolValue is_approved = 3;
 }
 
+// DeleteUserRequest is a request to delete a user. This request must be made with the user's
+// credentials.
```

Comment:
maybe add a comment that it deletes the org if the user is the last one in the org